README.md

```diff
@@ -271,6 +272,34 @@ interface HelloWorldView : TiView {
 ``` 
 Back in the Java days we had to use `it` inside the `sendToView`-lambda.
 
+#### Coroutines
+If you're using Kotlin's Coroutines we offer a `CoroutineScope` that scopes to a presenter's lifecycle.
+
+```kotlin
+class HelloWorldPresenter : TiPresenter<HelloWorldView> {
+
+  private val scope = TiCoroutineScope(this, Dispatchers.Default)
+
+  override fun onCreate() {
+      scope.launch { ... }
+  }
+}
+```
+The created `Job` will automatically be cancelled when the presenter is destroyed.
+
+Alternatively, you can launch jobs that get cancelled when a `TiView` detaches:
+```kotlin
+class HelloWorldPresenter : TiPresenter<HelloWorldView> {
+
+  private val scope = TiCoroutineScope(this, Dispatchers.Default)
+
+  override fun onAttachView(view: HelloWorldView) {
+      scope.launchUntilViewDetaches { ... }
+  }
+}
+```
+However, be careful that `launchUntilViewDetaches` can only be called when there is a view attached!
```

Comment:
Can't we do it handle similar to the `deliverToView`? 🤔 
I mean currently it is like the `viewOrThrow`.
But in some cases we don't know if there is a view attached. Means, we call - for safety reasons - `deliverToView`.
Means we have to wrap the `launchUntilViewDetaches` into `deliverToView` to make sure its not crashing.
That is a bad behaviour and looks odd.

When we can handle it similar to `deliverToView` - execute when there is an view; otherwise wait until a view got attached - we don't have to wrap this into `deliverToView`

Reply:
No, what you describe is wrong. It has nothing to do with `viewOrThrow` or `deliverToView`. 

This should behave similar to [manageViewDisposable](https://github.com/grandcentrix/ThirtyInch/blob/master/thirtyinch-rx2/src/main/java/net/grandcentrix/thirtyinch/rx2/RxTiPresenterDisposableHandler.java#L95), which throws when there is now view. So, for the sake of consistency I made it the same.